wikipron/scrape.py

```diff
@@ -14,16 +14,19 @@
 _PAGE_TEMPLATE = "https://en.wiktionary.org/wiki/{word}"
 
 
-def _skip_word(word: str) -> bool:
-    # Skips multiword examples.
-    if " " in word:
-        return True
+def _skip_word(word: str, no_skip_space: bool) -> bool:
     # Skips examples containing a dash.
     if "-" in word:
         return True
     # Skips examples containing digits.
     if re.search(r"\d", word):
         return True
+    if no_skip_space:
+        if " " in word:
```

Comment:
you don't need this inner check: it can simply be:

`if no_skip_space: return False`.

Or better yet you'd replace the entire block 24-29 with something like:

```
if not no_skip_space and " " in word: return True
else: return False
```